docs/dev/json-api.md

```diff
@@ -0,0 +1,65 @@
+# JSON:API
+
+## Modules
+
+We have enabled and configured the core Drupal [`jsonapi`](https://www.drupal.org/docs/core-modules-and-themes/core-modules/jsonapi-module/api-overview) module specifically to allow uploading of WFO daily situation reports, which are produced as PDFs.
+
+We are also enabling the following core Drupal modules:
+
+- [`basic_auth`](https://www.drupal.org/docs/8/core/modules/basic_auth/overview) for authentication
+- [`jsonapi_extras`](https://www.drupal.org/project/jsonapi_extras) by default, prevent resources (and resource fields) from being accessible via the API
+- [`jsonapi_defaults`](https://www.drupal.org/project/jsonapi_defaults) is a submodule of the above which adds defaults to API resource types
```

Comment:
Do you think we should remove the link here? I wasn't paying attention to the second half of this line and clicked it. Then I got worried because it links to a deprecated, unsupported module. 😂 Took me a couple minutes to realize what happened.

Reply:
Good idea. It's weird because `jsonapi_defaults` is a submodule... not sure why they didn't just fold it in outright. Oh well. Fixed.